lib/Doctrine/ORM/Proxy/Factory/StaticProxyFactory.php

```diff
@@ -121,7 +121,8 @@ private function makeInitializer(ClassMetadata $metadata, EntityPersister $persi
             $persister
 ) : bool {
             $originalInitializer = $initializer;
-            $initializer         = null;
+
+            $initializer = null;
```

Comment:
Hmm, what happened here?

Reply:
Looks like a bug in SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountAfterControlStructure

FILE: lib/Doctrine/ORM/Proxy/Factory/StaticProxyFactory.php
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 111 | ERROR | [x] Expected 1 lines after "return", found 0.
     |       |     (SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Reply:
Also run into the issue - upstream issue is open https://github.com/slevomat/coding-standard/issues/635